class InputError(Exception):
    pass

class QuitError(Exception):
    pass
